### ISSUE TITLE
Full UI: EndExplorationInteractionView  [Blocked: #15, #157]

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#35]
This is tracking implementing the fully polished UI version of the end exploration interaction.

See the web frontend for details on how the end exploration interaction works.

Mock: https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/40d6cf5f-f673-42c8-af15-035f711ef6b9/PM-IN-Introduction-

## Checklist
- [ ] Return to Topic Button
<img width="328" alt="Screenshot 2019-11-08 at 3 51 33 PM" src="https://user-images.githubusercontent.com/9396084/68469514-55de0e80-0240-11ea-915b-7ea9980997e9.png">

- [ ] Final touch up with padding, margin, animations and Accessibility Scanner test

**Comment by BenHenning** [comment]
This is blocked on #28 to introduce the general interaction architecture. This also likely is not needed for practice sessions since it's exploration-specific, or it needs to be generalized to handle the end of both explorations and practice sessions (or we need a separate interaction to handle the practice sessions case).

**Comment by BenHenning** [comment]
This is blocked on #15 for a full domain controller implementation and #157 for the initial structure introduction for this interaction.

**Comment by BenHenning** [comment]
@rt4914 can you clarify what remaining work needs to be done here?

**Comment by rt4914** [comment]
The UI is ready but the final implementation depends on #377

**Comment by rt4914** [comment]
Closed via #1653
